EPA/project.md

```diff
@@ -0,0 +1,67 @@
+# Work-Based Project FAQs
+
+Frequently asked questions from learners who have started EPA.
+
+This compliments the "EPA Submission Pack" which you should have received when you started EPA. Please contact the trainers if you do not have this pack.
+
+<details markdown="1">
+<summary markdown="1">
+Is there anything else I should do while I'm working on the project?
+</summary>
+
+It is worth keeping some notes as you go to make it easier for you to fill in the Declaration Form and put together your evidence and presentation.
+In particular, keep notes of:
+
+* What you’ve worked on
+* Roughly how long you’ve spent on-project each week
+* How you have demonstrated each KSB + assessment criteria, and which ones you still need to demonstrate
```

Comment:
Could suggest tracking progress on the project plan mapping document itself, ticking off each KSB as it's covered while making adjustments for project changes.

Reply:
Good idea